deploy/helm/magda/charts/storage-api/templates/deployment.yaml

```diff
@@ -39,6 +39,11 @@ spec:
             secretKeyRef:
               name: storage-secrets
               key: accesskey
+        - name: JWT_SECRET
+          valueFrom:
+            secretKeyRef:
+              name: storage-secrets
+              key: jwt-secret
```

Comment:
The JWT secret is already in `auth-secrets`, can we just use that one?

Reply:
Yep, sure can.

Reply:
Yep